lib/core/repository/cms/cms.rb

```diff
@@ -7,12 +7,19 @@ def initialize(options = {})
       end
 
       def find(id)
-        response = connection.get('%{locale}/%{id}.json' % { locale: I18n.locale, id: id })
+        resource_url = "%{locale}/%{page_type}/%{id}.json" % { locale: I18n.locale, page_type: resource_name, id: id }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.